### ISSUE TITLE
Email-notifier unit tests are currently failing

### ISSUE THREAD

**Issue by mdebarros** [mojaloop/project#626]
**Summary**:
Email-notifier unit tests are currently failing

```BASH
$ npm run test

> email-notifier@5.1.0 test /Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier
> nyc mocha test/unit/**/*.test.js --timeout=4000



  0 passing (1ms)

TAP version 13
# nodeMailer unit tests (sendMail.js) :
# RxJs Observable Tests (Action Observable) :
# Templates unit tests (Index.js) :
# nodeMailer unit tests (sendMail.js) :   sendMail should return success.
else hit
ok 1 should be equivalent
# nodeMailer unit tests (sendMail.js) :   sendMail should throw an error.
ok 2 some error
# RxJs Observable Tests (Action Observable) :  Should test the actionObservable succeeds
ok 3 should be equivalent
# RxJs Observable Tests (Action Observable) :  Should fail when sent mail message fails.
(node:38963) UnhandledPromiseRejectionWarning: Error: Failure
    at actionTest.test (/Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/test/unit/observables/action.test.js:216:37)
    at /Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/tapes/index.js:69:9
    at /Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/tapes/index.js:82:7
    at /Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/tapes/node_modules/async/lib/async.js:52:16
    at Object.<anonymous> (/Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/tapes/node_modules/async/lib/async.js:269:32)
    at Object.end (/Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/tapes/node_modules/async/lib/async.js:44:16)
    at actionTest.beforeEach.t (/Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/test/unit/observables/action.test.js:44:7)
    at /Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/tapes/index.js:91:5
    at /Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/tapes/node_modules/async/lib/async.js:181:20
    at iterate (/Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/tapes/node_modules/async/lib/async.js:262:13)
    at Object.async.forEachOfSeries.async.eachOfSeries (/Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/tapes/node_modules/async/lib/async.js:281:9)
    at Object.async.forEachSeries.async.eachSeries (/Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/tapes/node_modules/async/lib/async.js:214:22)
    at runWrapperFns (/Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/tapes/index.js:90:9)
    at Test.<anonymous> (/Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/tapes/index.js:81:5)
    at Test.bound [as _cb] (/Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/tape/lib/test.js:92:32)
    at Test.run (/Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/tape/lib/test.js:111:10)
    at Test.bound [as run] (/Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/tape/lib/test.js:92:32)
    at Immediate.next (/Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/tape/lib/results.js:75:19)
    at runCallback (timers.js:705:18)
    at tryOnImmediate (timers.js:676:5)
    at processImmediate (timers.js:658:5)
(node:38963) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 3)
(node:38963) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
not ok 4 test exited without ending
  ---
    operator: fail
    at: process.<anonymous> (/Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/tape/index.js:90:19)
    stack: |-
      Error: test exited without ending
          at Test.assert [as _assert] (/Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/tape/lib/test.js:224:54)
          at Test.bound [as _assert] (/Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/tape/lib/test.js:92:32)
          at Test.fail (/Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/tape/lib/test.js:315:10)
          at Test.bound [as fail] (/Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/tape/lib/test.js:92:32)
          at Test._exit (/Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/tape/lib/test.js:189:14)
          at Test.bound [as _exit] (/Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/tape/lib/test.js:92:32)
          at process.<anonymous> (/Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/tape/index.js:90:19)
          at process.emit (events.js:194:15)
          at processEmit [as emit] (/Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/nyc/node_modules/signal-exit/index.js:149:35)
  ...
not ok 5 test exited without ending
  ---
    operator: fail
    at: process.<anonymous> (/Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/tape/index.js:90:19)
    stack: |-
      Error: test exited without ending
          at Test.assert [as _assert] (/Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/tape/lib/test.js:224:54)
          at Test.bound [as _assert] (/Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/tape/lib/test.js:92:32)
          at Test.fail (/Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/tape/lib/test.js:315:10)
          at Test.bound [as fail] (/Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/tape/lib/test.js:92:32)
          at Test._exit (/Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/tape/lib/test.js:189:14)
          at Test.bound [as _exit] (/Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/tape/lib/test.js:92:32)
          at process.<anonymous> (/Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/tape/index.js:90:19)
          at process.emit (events.js:194:15)
          at processEmit [as emit] (/Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/nyc/node_modules/signal-exit/index.js:149:35)
  ...
not ok 6 test exited without ending
  ---
    operator: fail
    at: process.<anonymous> (/Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/tape/index.js:90:19)
    stack: |-
      Error: test exited without ending
          at Test.assert [as _assert] (/Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/tape/lib/test.js:224:54)
          at Test.bound [as _assert] (/Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/tape/lib/test.js:92:32)
          at Test.fail (/Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/tape/lib/test.js:315:10)
          at Test.bound [as fail] (/Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/tape/lib/test.js:92:32)
          at Test._exit (/Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/tape/lib/test.js:189:14)
          at Test.bound [as _exit] (/Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/tape/lib/test.js:92:32)
          at process.<anonymous> (/Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/tape/index.js:90:19)
          at process.emit (events.js:194:15)
          at processEmit [as emit] (/Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/nyc/node_modules/signal-exit/index.js:149:35)
  ...
not ok 7 test exited without ending
  ---
    operator: fail
    at: process.<anonymous> (/Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/tape/index.js:90:19)
    stack: |-
      Error: test exited without ending
          at Test.assert [as _assert] (/Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/tape/lib/test.js:224:54)
          at Test.bound [as _assert] (/Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/tape/lib/test.js:92:32)
          at Test.fail (/Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/tape/lib/test.js:315:10)
          at Test.bound [as fail] (/Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/tape/lib/test.js:92:32)
          at Test._exit (/Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/tape/lib/test.js:189:14)
          at Test.bound [as _exit] (/Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/tape/lib/test.js:92:32)
          at process.<anonymous> (/Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/tape/index.js:90:19)
          at process.emit (events.js:194:15)
          at processEmit [as emit] (/Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/nyc/node_modules/signal-exit/index.js:149:35)
  ...
not ok 8 test exited without ending
  ---
    operator: fail
    at: process.<anonymous> (/Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/tape/index.js:90:19)
    stack: |-
      Error: test exited without ending
          at Test.assert [as _assert] (/Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/tape/lib/test.js:224:54)
          at Test.bound [as _assert] (/Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/tape/lib/test.js:92:32)
          at Test.fail (/Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/tape/lib/test.js:315:10)
          at Test.bound [as fail] (/Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/tape/lib/test.js:92:32)
          at Test._exit (/Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/tape/lib/test.js:189:14)
          at Test.bound [as _exit] (/Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/tape/lib/test.js:92:32)
          at process.<anonymous> (/Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/tape/index.js:90:19)
          at process.emit (events.js:194:15)
          at processEmit [as emit] (/Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/nyc/node_modules/signal-exit/index.js:149:35)
  ...
not ok 9 test exited without ending
  ---
    operator: fail
    at: process.<anonymous> (/Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/tape/index.js:90:19)
    stack: |-
      Error: test exited without ending
          at Test.assert [as _assert] (/Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/tape/lib/test.js:224:54)
          at Test.bound [as _assert] (/Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/tape/lib/test.js:92:32)
          at Test.fail (/Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/tape/lib/test.js:315:10)
          at Test.bound [as fail] (/Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/tape/lib/test.js:92:32)
          at Test._exit (/Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/tape/lib/test.js:189:14)
          at Test.bound [as _exit] (/Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/tape/lib/test.js:92:32)
          at process.<anonymous> (/Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/tape/index.js:90:19)
          at process.emit (events.js:194:15)
          at processEmit [as emit] (/Users/mdebarros/Documents/ModusDocs/Projects/MojaLoop/git/fork/email-notifier/node_modules/nyc/node_modules/signal-exit/index.js:149:35)
  ...

1..9
# tests 9
# pass  3
# fail  6

-----------------|----------|----------|----------|----------|-------------------|
File             |  % Stmts | % Branch |  % Funcs |  % Lines | Uncovered Line #s |
-----------------|----------|----------|----------|----------|-------------------|
All files        |    53.11 |    13.21 |    36.36 |    52.52 |                   |
 src/lib         |    54.79 |        0 |        0 |    54.79 |                   |
  config.js      |      100 |      100 |      100 |      100 |                   |
  enum.js        |      100 |      100 |      100 |      100 |                   |
  utility.js     |     45.9 |        0 |        0 |     45.9 |... 17,318,319,321 |
 src/lib/kafka   |    22.89 |        0 |        0 |    22.89 |                   |
  consumer.js    |    23.08 |        0 |        0 |    23.08 |... 36,137,139,140 |
  index.js       |      100 |      100 |      100 |      100 |                   |
  producer.js    |    17.07 |        0 |        0 |    17.07 |... 10,125,126,128 |
 src/nodeMailer  |      100 |      100 |      100 |      100 |                   |
  sendMail.js    |      100 |      100 |      100 |      100 |                   |
 src/observables |       75 |        0 |       50 |       75 |                   |
  actions.js     |       75 |        0 |       50 |       75 |... 27,128,129,131 |
 templates       |    83.33 |       50 |      100 |    81.48 |                   |
  index.js       |    83.33 |       50 |      100 |    81.48 |    17,19,31,33,42 |
-----------------|----------|----------|----------|----------|-------------------|
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! email-notifier@5.1.0 test: `nyc mocha test/unit/**/*.test.js --timeout=4000`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the email-notifier@5.1.0 test script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/mdebarros/.npm/_logs/2019-02-12T18_31_37_037Z-debug.log
```

**Severity**:
Medium

**Priority**:
Medium

**Expected Behavior**
Unit tests pass

**Steps to Reproduce**
  1. Pull master branch from email-notifier
  2. npm install
  3. npm run test

The issue occurs in both Node version 8.9.4 and 10.15.1.

**Specifications**
  - Component (if known): email-notifier
  - Version: 5.1.0
  - Platform: MacOs Mojave & Linux Ubuntu 16.4
  - Subsystem:
  - Type of testing:
  - Bug found/raised by: @mdebarros 

**Notes**:

- Severity when opened: Medium
- Priority when opened: Medium

**Comment by elnyry** [comment]
This should be addressed in #559

**Comment by elnyry** [comment]
Covered in #559, as that item is in review now, closing this
